vscode-dotnet-runtime-library/src/Acquisition/AcquisitionInvoker.ts

```diff
@@ -16,6 +16,7 @@ import {
     DotnetAcquisitionUnexpectedError,
     DotnetOfflineFailure,
 } from '../EventStream/EventStreamEvents';
+import { timeoutConstants } from '../Utils/Constants';

```

Comment:
I was patterning off another extension I was looking at. So you suggest defining constants in one of the files where they're used, even if they're used in many files? Or separate constant files for constants that are used together (reducing coupling between larger parts of the system)?

Reply:
> defining constants in one of the files where they're used, even if they're used in many files?

Define the constants where it is used. If it is shared, it should be considered just like referencing a method. Does it have to be shared? Why the other component need to reference it? And what the dependency layering should be.

What I don't like here especially is the constants are put in a same file just for the sake that they are constants. And everything depends on one thing that they don't need to.

Although if it is the existing style, it would be hard to move away from (half and half is also bad). Maybe move const to the file only when it is shared.